python/social_bot/util/replay_buffer.py

```diff
@@ -18,48 +18,59 @@ def __init__(self, buffer_size, history_length, future_length):
         self._buffer_size = buffer_size
         self._history_length = history_length
         self._future_length = future_length
-        self._buffer = deque(maxlen=buffer_size)
+        self._buffer = [None] * buffer_size
```

Comment:
implement its own cirque class, with [], append() method, would reduce the code change by a lot.

Reply:
Good idea.